packages/core/echo/echo/src/Type.ts

```diff
@@ -29,6 +29,7 @@ export const Relation = EchoSchema.EchoRelation;
  */
 export const Ref: <S extends Obj.Any>(schema: S) => EchoSchema.Ref$<Schema.Schema.Type<S>> = EchoSchema.Ref;
 
+// TODO(buurdon): Move to Obj?
```

Comment:
this api was deliberatly planned out - see readme. Lets do start all major changes from editing the table to keep consistency